src/main-mw.ts

```diff
@@ -18,7 +19,13 @@ export default function(): Middleware {
     }),
     problem(),
     session({
-      store: 'memory',
+      store: process.env.REDIS_SESSION ? new RedisStore({
+        prefix: 'A12N-session',
```

Comment:
Might be worthwhile to have this as a env var as well

Reply:
agreed, could be helpful if multiple environments use the same redis instance. Maybe something like REDIS_PREFIX ?